notation.go

```diff
@@ -191,14 +194,17 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, re
 	}
 
 	// passing nil signature to check 'skip'
+	logger.Info("Checking whether signature verification should be skipped or not")
 	outcome, err := verifier.Verify(ctx, ocispec.Descriptor{}, nil, opts)
 	if err != nil {
 		if outcome == nil {
 			return ocispec.Descriptor{}, nil, err
 		}
 	} else if reflect.DeepEqual(outcome.VerificationLevel, trustpolicy.LevelSkip) {
+		logger.Infoln("Verification skipped for", remoteOpts.ArtifactReference)
 		return ocispec.Descriptor{}, []*VerificationOutcome{outcome}, nil
 	}
+	logger.Info("Trust policy is not configured to skip signature verification")
```

Comment:
I feel like we still need to add a `Check over.` in the beginning of this log. This is because, every time we check `skip`, there will be an integrity ERRO printed out. New users might be confused. Explicitly saying `check over` mitigates users' concerns.
```suggestion
	logger.Info("Check over. Trust policy is not configured to skip signature verification")
```

Reply:
Updated.